### ISSUE TITLE
ImportError: cannot import name 'Backend'

### ISSUE THREAD

**Issue by leaviljanen** [tcosolutions/betterscan#205]
I am getting an error on the latest on VMWare guest Debian 12.
The error is the same whether I use the sh < curl  or run the worker-cli manually

```
user@debian:~/projects/ttt/src$ export CODE_DIR=/home/user/projects/ttt/src
user@debian:~/projects/ttt/src$ sh <(curl https://raw.githubusercontent.com/tcosolutions/betterscan-ce/main/cli.sh)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   943  100   943    0     0   2813      0 --:--:-- --:--:-- --:--:--  2823
Traceback (most recent call last):
  File "/usr/local/bin/checkmate", line 5, in <module>
    from checkmate.scripts.manage import main
  File "/usr/local/lib/python3.8/site-packages/checkmate/scripts/manage.py", line 23, in <module>
    from checkmate.management import helpers, commands
  File "/usr/local/lib/python3.8/site-packages/checkmate/management/helpers.py", line 12, in <module>
    from checkmate.lib.backend import Backend
  File "/usr/local/lib/python3.8/site-packages/checkmate/lib/backend.py", line 2, in <module>
    from blitzdb.backends.sql import Backend as BlitzDBSQLBackend
  File "/usr/local/lib/python3.8/site-packages/blitzdb/backends/sql/__init__.py", line 22, in <module>
    from .backend import Backend
ImportError: cannot import name 'Backend' from 'blitzdb.backends.sql.backend' (/usr/local/lib/python3.8/site-packages/blitzdb/backends/sql/backend.py)
```

```
user@debian:~/projects/ttt/src$ sudo docker run -e CODE_DIR=$CODE_DIR -e OPENAI_GPT_API -v ${CODE_DIR}:${CODE_DIR} -ti tcosolutions/betterscan-worker-cli:latest /bin/sh -c "cd $CODE_DIR && git config --global --add safe.directory $CODE_DIR && checkmate init --backend sqlite --backend-opts \"sqlite:////${CODE_DIR}/.checkmate/database.db\""
Traceback (most recent call last):
  File "/usr/local/bin/checkmate", line 5, in <module>
    from checkmate.scripts.manage import main
  File "/usr/local/lib/python3.8/site-packages/checkmate/scripts/manage.py", line 23, in <module>
    from checkmate.management import helpers, commands
  File "/usr/local/lib/python3.8/site-packages/checkmate/management/helpers.py", line 12, in <module>
    from checkmate.lib.backend import Backend
  File "/usr/local/lib/python3.8/site-packages/checkmate/lib/backend.py", line 2, in <module>
    from blitzdb.backends.sql import Backend as BlitzDBSQLBackend
  File "/usr/local/lib/python3.8/site-packages/blitzdb/backends/sql/__init__.py", line 22, in <module>
    from .backend import Backend
ImportError: cannot import name 'Backend' from 'blitzdb.backends.sql.backend' (/usr/local/lib/python3.8/site-packages/blitzdb/backends/sql/backend.py)

```

As if the sqlite directive isn't really sticking, blitzdb isn't sqlite? I have no clue, have fiddled with creating checkmate config.json manually etc but no luck.

**Comment by marcinguy** [comment]
@leaviljanen i think the imports in blitzdb is messed up. Made a quick commit on mobile (cannot find my MacBook Pro)

Blitzdb is a database to store documents with revisions in json format using sqlalchemy to support  many backend ie memory, sqllite, various SQL databases. U can read more on blitztdo repo code

Please try now

Pr are also welcome 🙏

**Comment by leaviljanen** [comment]
Unfortunately the fix didn't succeed, the current container is still unusable.
```
user@debian:~/projects/ttt/src$ docker images
REPOSITORY                           TAG                     IMAGE ID       CREATED             SIZE
tcosolutions/betterscan-worker-cli   latest                  207094cb26f0   About an hour ago   1.59GB
sunsolution/betterscan-worker-cli    latest                  bfb25af01410   7 days ago          1.59GB
rabbitmq                             3.8-management-alpine   69b0f3f87ca9   2 years ago         166MB
postgres                             13.2                    82b8b88e26bc   3 years ago         314MB
user@debian:~/projects/ttt/src$ sh <(curl https://raw.githubusercontent.com/tcosolutions/betterscan-ce/main/cli.sh)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   943  100   943    0     0   6256      0 --:--:-- --:--:-- --:--:--  6328
Traceback (most recent call last):
  File "/usr/local/bin/checkmate", line 5, in <module>
    from checkmate.scripts.manage import main
  File "/usr/local/lib/python3.8/site-packages/checkmate/scripts/manage.py", line 23, in <module>
    from checkmate.management import helpers, commands
  File "/usr/local/lib/python3.8/site-packages/checkmate/management/helpers.py", line 12, in <module>
    from checkmate.lib.backend import Backend
  File "/usr/local/lib/python3.8/site-packages/checkmate/lib/backend.py", line 2, in <module>
    from blitzdb.backends.sql import Backend as BlitzDBSQLBackend
  File "/usr/local/lib/python3.8/site-packages/blitzdb/backends/sql/__init__.py", line 22, in <module>
    from .backend import Backend
ImportError: cannot import name 'Backend' from 'blitzdb.backends.sql.backend' (/usr/local/lib/python3.8/site-packages/blitzdb/backends/sql/backend.py)
```

(Right now I'm on a deadline so can't drill deep into the mysteries of this to be able to provide a PR, sorry)

**Comment by marcinguy** [comment]
@leaviljanen thanks for your reply and debug/fix efforts

Yea I noticed that models in checkmate were missing a field. And some code in helpers was not correct. Improved it also to be better (ai code review had good hints/solutions and helpful, especially if your dev skills are rusty)

I triggered a rebuild 

Not sure what u did but last time I think I fixed it so it should be good now

https://github.com/tcosolutions/blitzdb3/blob/ff3b597f34822e1139948c8dcc5decdbaf1d1bce/blitzdb/__init__.py#L22

Above it is imported 

Let me know if it works. On a mobile now and cannot test
